src/Resources/config/doctrine/AuditLog.orm.xml

```diff
@@ -7,6 +7,8 @@
         <id name="id" type="bigint" column="id">
             <generator strategy="AUTO" />
         </id>
+        <field name="ip" nullable="true" type="string" length="32"/>
```

Comment:
IPv6 has a max length of 39. That being said, I personally prefer using the max length of the column type. I'd prefer to use 255 here.